### PR TITLE
New Article "What's next in AdonisJS6"

### PR DESCRIPTION
**Pull request by Julien-R44**
Added more stuff and technical details about new @adonisjs/vite

Maybe we can merge and publish it the Friday 26

Lemme know if looks okay to you

### CODE REVIEW

**Review by thetutlage — CHANGES_REQUESTED** [review]

File: content/blog_posts/articles/whats-coming-next-adonisjs-6.md [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+Last year, we've spent a lot of time testing new ideas and features. Some of them are already in pretty good state, while others are still being thought and worked on. In this article, I'm going to share some of those ideas.
+
+**Please, do not expect or ask for any release dates or timelines. We will share more details when we are ready.**
+
+## Migrating last few packages to AdonisJS 6
+
+First focus will be to migrate the last few packages to AdonisJS 6. We can't commit on a specific date, but expect them to be migrated in the next few weeks :
+
+- Drive
+- Limiter
```
Comment (thetutlage) [inline content/blog_posts/articles/whats-coming-next-adonisjs-6.md:18]: Let's add the link to release notes here. Saying "Limiter (Already released)" https://github.com/adonisjs/limiter/releases/tag/v2.1.0

**Review by thetutlage — CHANGES_REQUESTED** [review]

File: content/blog_posts/articles/whats-coming-next-adonisjs-6.md [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+- Lucid Slugify
+- Attachment lite
+- Route model binding
+- Health checks - Removed from core to be its own package.
+
+## @adonisjs/cache
+
+This summer we worked on [BentoCache](bentocache.dev) - a caching library for Node.js. It is a standalone and can be used with any framework. We are planning to have a first-party integration with AdonisJS 6, and so with its own set of features.
```
Comment (thetutlage) [inline content/blog_posts/articles/whats-coming-next-adonisjs-6.md:26]: Full domain with `https://bentocache.dev` is needed here.

Comment (thetutlage) [inline content/blog_posts/articles/whats-coming-next-adonisjs-6.md:26]: "This summer" should be "Last summer (2023)"

File: content/blog_posts/articles/whats-coming-next-adonisjs-6.md [hunk 3]
```diff
@@ -0,0 +1,232 @@
...
+
+const transmit = new Transmit({ baseUrl: 'localhost:3333' })
+transmit.listen('chat', (message) => console.log(message))
+```
+
+Transmit should be released in the next few weeks. Get ready for it!
+
+## @adonisjs/locks
+
+A few weeks ago, we released [Verrou](verrou.dev), an agnostic package for NodeJS that can be used with any framework. The purpose of the package is to provide a simple API for locking (mutexes), also known as "Atomic locks". Verrou supports several drivers: Redis, Postgre, Mysql, Sqlite, In-memory ...
```
Comment (thetutlage) [inline content/blog_posts/articles/whats-coming-next-adonisjs-6.md:194]: Http for `verrou.dev`

**Review by thetutlage — CHANGES_REQUESTED** [review]

File: content/blog_posts/articles/whats-coming-next-adonisjs-6.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (Julien-R44) [reply content/blog_posts/articles/whats-coming-next-adonisjs-6.md]: Thanks !

**Review by thetutlage — APPROVED** [review]

**Pull request merged**